src/lib/layout-gap-attributes.scss

```diff
@@ -0,0 +1,56 @@
+@use "sass:list";
+
+@import "mixins";
+
+/// Includes selectors for attributes which can be used to add a specific gap to elements in a flex container
+///
+/// @param {number} $value
+///   Will start generating selectors for $value $gapUnits
+/// @param {string} $gapUnits
+///   Will generate selectors for the units passed in $gapUnits, e.g. px em rem
+/// @param {string} $name
+///   Will generate selectors for a specific media size, e.g. xs, sm, md, ...
+@mixin layout-gap-attribute($value, $gapUnits, $name: false) {
+	@each $unit in $gapUnits {
+		*[data-layout-gap#{if($name, '-' + $name, '')}="#{$value}#{$unit}"] {
+			gap: #{$value}#{$unit};
+		}
+	}
+}
+
+/// Includes selectors for attributes which can be used to add a gap to elements in a flex container
+///
+/// @param {number} $from
+///   Will start generating selectors from $from $gapUnits, $from can also be used as a value list to generate value $gapUnits for every value in $from
+/// @param {number} $to
+///   Will generate selectors until including $to $gapUnits
+/// @param {string} $gapUnits
+///   Will generate selectors for the units passed in $gapUnits, e.g. px em rem
+/// @param {bool} $includeSelectorsForMediaSizes
+///	  Will generate additional selectors for flex layout media sizes
+@mixin layout-gap-attributes($from: 1, $to: 24, $gapUnits: px em rem, $includeSelectorsForMediaSizes: true) {
```

Comment:
To be honest I think the double functionality of the `$from` parameter is difficult, although it keeps the API unchanged while extending the functionality. What do you think about adding a second mixin?

Reply:
Agreed. What do you suggest to name the list mixins?

Reply:
Maybe something as simple as `layout-gap-attributes-list` or variations of this with `-as-list` or `-fixed-list` or `-with-values`?

I think I like the `-as-list` suffix the most. What do you think?

Reply:
Yes, that's also good.

Reply:
Ok I'll do that.